fmt.sh

```diff
@@ -112,24 +125,21 @@ run dade-copyright-headers "$dade_copyright_arg" .
 
 # We do test hlint via Bazel rules but we run it separately
 # to get linting failures early.
-HLINT=(hlint -j4)
-echo "\$ ${HLINT[*]}"
-if [ "$hlint_diff" = "true" ]; then
-  #  We do not run on deleted files, or files that have been added since we last rebased onto trunk.
-  changed_haskell_files="$(git diff --name-only --diff-filter=ACMRT "$merge_base" | grep '\.hs$' || [[ $? == 1 ]])"
-  if [[ -n "$changed_haskell_files" ]]; then
-    "${HLINT[@]}" -j4 $changed_haskell_files
-  fi
+if [ "$diff_mode" = "true" ]; then
+  check_diff $merge_base '\.hs$' hlint -j4
+  check_diff $merge_base '\.java$' javafmt "${javafmt_args[@]:-}"
 else
-  "${HLINT[@]}" --git
+  run hlint -j4 --git
+  java_files=$(find . -name "*.java")
+  if [[ -z "$java_files" ]]; then
+    echo "Unexpected: no Java file in the repository"
```

Comment:
accepting bets for when we’ll reach that.

Reply:
Do you have a Daml driven betting platform? 😉